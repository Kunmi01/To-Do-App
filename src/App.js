import React from 'react';
import ActionRecorderContainer from './components/ActionRecorder';
import ToDoListContainer from './components/ToDoList';
import './App.scss';

const App = () => {
  return (
    <div className="app">
      <ActionRecorderContainer />
      <ToDoListContainer />
    </div>
  );
};

export default App;
