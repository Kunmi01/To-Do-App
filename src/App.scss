@import './scss/base.scss';

.app {
  display: flex;
  flex-direction: column;
}
