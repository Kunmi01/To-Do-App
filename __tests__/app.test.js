import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../src/App';

describe('App', () => {
  const wrapper = shallow(<App />);

  it('renders DUMB component', () => {
    expect(wrapper).toHaveLength(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
