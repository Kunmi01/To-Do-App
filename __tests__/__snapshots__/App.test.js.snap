// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders DUMB component 1`] = `
<div
  className="app"
>
  <Connect(ActionRecorder) />
  <Connect(ToDoList) />
</div>
`;
